image/templates/helm/shared/templates/_metadata.tpl

```diff
@@ -1,73 +1,81 @@
 {{/*
-  srox.labels $ $objType $objName [ $labels ]
+  srox.labels $ $objType $objName [ $extraLabels ]
 
-  Format labels for $objType/$objName as YAML.
+  Format labels for $objType/$objName as YAML. This takes into consideration the $extraLabels,
+  if provided, plus labels added using the generic `customize` configuration mechanism.
+  Note that provided $eextraLabels can be modified via `customize`.
```

Comment:
```suggestion
  Note that provided $extraLabels can be modified by the user via `customize`.
```

Reply:
Also, although this is not strictly in scope of this PR, but since you're editing the description anyway, would you mind adding a note here (and maybe in `srox.podLabels` too) on when/why it's appropriate to use one over the other?

Reply:
done